src/OpalCompiler-Tests/MethodPragmaTest.class.st

```diff
@@ -240,9 +330,8 @@ MethodPragmaTest >> testMethodSelector [
 
 { #category : 'tests - compiled' }
 MethodPragmaTest >> testNoPragma [
-	| method |
-	method := self compile: '' selector: #foo.
-	self assert: method pragmas equals: #()
+
+	OpalCompiler new parse: self methodNoPragma
```

Comment:
This test has no assertion now. Is that normal?

Reply:
removed